File: src/main/java/tech/zolhungaj/amqapi/servercommands/globalstate/AvatarDriveUpdate.java
package tech.zolhungaj.amqapi.servercommands.globalstate;

import tech.zolhungaj.amqapi.servercommands.CommandType;
import tech.zolhungaj.amqapi.servercommands.objects.AvatarDriveContributionCombo;
import tech.zolhungaj.amqapi.servercommands.objects.AvatarDriveNomination;

import java.util.List;

@CommandType("avatar drive changes")
public record AvatarDriveUpdate(
        List<AvatarDriveNomination> top5Nominations,
        List<AvatarDriveContributionCombo> recent8
){}

File: src/main/java/tech/zolhungaj/amqapi/servercommands/globalstate/AvatarDriveUpdate.kt
package tech.zolhungaj.amqapi.servercommands.globalstate

import tech.zolhungaj.amqapi.servercommands.CommandType
import tech.zolhungaj.amqapi.servercommands.objects.AvatarDriveContributionCombo
import tech.zolhungaj.amqapi.servercommands.objects.AvatarDriveNomination

@JvmRecord
@CommandType("avatar drive changes")
data class AvatarDriveUpdate(
    val top5Nominations: List<AvatarDriveNomination>,
    val recent8: List<AvatarDriveContributionCombo>
)
